### ISSUE TITLE
block triggering default select action

### ISSUE THREAD

**Issue by mogui** [spotify/HubFramework#297]
I don't know if it is an issue or I miss something I couldn't find in the documentation.
What I want to accomplish is adding a long tap gesture to an HUBComponent in addition to the single tap default behaviour.
The problem is that I couldn't find a way to prevent the default action to be triggered when the long tap gesture is handled, they both fire. I've tried in many ways but I have not succeded the default tap is always triggered.
How can it be done?

**Comment by mhallendal** [comment]
When you say default action, which one do you mean. The action handling is pretty hard to follow and I'm still trying to wrap my head around how it's handled. We have been looking into replacing it with a simpler solution and will likely do that at some point but we haven't had time to work on it yet.

If you could describe a bit more about the use case I might be able to help you out.

**Comment by mogui** [comment]
Hi ok,
I figured out a solution, but I'll explain the case anyway.

I've got a component (ie. a simple Image) and I want it to have to kind of actions:
1. when i tap on the image do Action1
2. when I long tap the image do Action2

The framwork allow me to do easily case 1. just by adding the action in the target node of the json configuring the component, BUT to implement the case 2. I had to add to the component a gesture recognizer.

The issue is that when I handle case 2. also case 1. is fired becaus case 1. is implemented inside the framework also with a gesture recognizer but at another view level so the two cannot syncronize.

What I was asking was if this was addressed in a way by the framework.

My solution for now is to make to implement myself both cases with a UIButton over the image and a gesture recognizer and triggering the actions by myself, bypassing completely the framework default.

I hope to have been complete in the explanation :)
